superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -97,6 +140,18 @@ def set_path_to_psf(self,path=None):
             if not os.path.exists(self.psf_path):
                 os.mkdir(self.psf_path)
 
+    def set_mask(self,mask_name='mask.fits',mask_dir=None):
+        if mask_dir is None:
+            self.mask_path = './mask_files'
```

Comment:
For the `set_mask()` and `set_weight()` functions, does it make more sense to set the default paths to be `os.path.join(self.work_dir, {mask/weight}_files)`?